geopandas/tests/test_sindex.py

```diff
@@ -151,6 +151,14 @@ def setup_method(self):
         self.df = GeoDataFrame(data, geometry="location")
         self.expected_size = len(data["location"])
 
+    #
+    def test_scattered_empty(self):
+        """Tests building sindex with interlived empty geometries.
```

Comment:
Typo: `interlived` => `interleaved`

Maybe this could just be `test_empty_geometries`?

Reply:
I wanted to differentiate it from existing empty geometry tests that check a single empty geometry (`test_empty_point`) or have empty geometries as input geometries (`test_query_empty_geometry`).

How about `test_empty_geomtries_in_tree`?

Reply:
`test_empty_tree_geometries`?  Understood about need to differentiate, not super picky about the name so long as it expresses what is actually being tested...

Reply:
Yup, sounds good. Pushed that name and fixed the typo.